01/app.js

```diff
@@ -1 +1,11 @@
-console.log('DOM');
\ No newline at end of file
+console.log("DOM");
+
+const elements = document.querySelector(
```

Comment:
Jest to jeden element, a nie wiele więc nazwa zmiennej wprowadza w błąd - powinno być bez `s`